homeassistant/components/tedee/views.py

```diff
@@ -0,0 +1,40 @@
+"""Views for Tedee integration, to be used for webhook from Tedee Bridge."""
+
+from http import HTTPStatus
+import logging
+
+from aiohttp import web
+
+from homeassistant.components.http.view import HomeAssistantView
+
+from .coordinator import TedeeApiCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TedeeWebhookView(HomeAssistantView):
+    """Handle Tedee Webhook requests."""
+
+    url = "/api/tedee/webhook"
+    name = "api:tedee:webhook"
+
+    requires_auth = True
```

Comment:
I'd like to have an opinion here please: Should I leave that endpoint authenticated only (which requires to send a long lived access token unencrypted every time, which grants access to the entire HA instance), or should I make it unauthenticated (it is only used to update status information anyways)?